contracts/0.8.25/vaults/VaultHub.sol

```diff
@@ -69,6 +69,11 @@ contract VaultHub is PausableUntilWithRoles {
         int256 inOutDelta;
         bytes32 withdrawalCredentials;
         uint256 liabilityShares;
+        uint96 shareLimit;
```

Comment:
not sure but might embed VaultSocket field right here instead (not a strong opinion though)